EIPS/eip-3475.md

```diff
@@ -260,47 +260,59 @@ function isApprovedFor(address _owner, address _operator) external view returns
 * @notice Issue MUST trigger when Bonds are issued. This SHOULD not include zero value Issuing.
 * @dev This SHOULD not include zero value issuing.
 * @dev Issue MUST be triggered when the operator (i.e Bank address) contract issues bonds to the given entity.
+* eg: emit Issue(_operator, 0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef,[IERC3475.Transaction(1,14,500)]); 
+* issue by address(operator) 500 DBIT-USD Bond(nonce14,class 1) to address 0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef.
 */
 event Issue(address indexed _operator, address indexed _to, Transaction[] _transactions); 
 // eg:
  
 emit Issue(_operator, 0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef,[IERC3475.Transaction(1,14,500)]); 
 // issue by address(operator) 500 DBIT-USD Bond(nonce14,class 0) to address 0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef.
-
 /** 
 * Redeem
 * @notice Redeem MUST trigger when Bonds are redeemed. This SHOULD not include zero value redemption.
+*e.g: emit Redeem(0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef,0x492Af743654549b12b1B807a9E0e8F397E44236E,[IERC3475.Transaction(1,14,500)]);
+* emit event when 5000 bonds of class 1, nonce 14 owned by address 0x492Af743654549b12b1B807a9E0e8F397E44236E are being redeemed by 0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef.
 */
 event Redeem(address indexed _operator, address indexed _from, uint256 classId, uint256 nonceId, uint256 _amount);
-//e.g: 
-emit Redeem(0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef,0x492Af743654549b12b1B807a9E0e8F397E44236E,[IERC3475.Transaction(1,14,500)]);
+
+//e.g: emit Redeem(0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef,0x492Af743654549b12b1B807a9E0e8F397E44236E,[IERC3475.Transaction(1,14,500)]);
 //emit event when 5000 bonds of class 1, nonce 14 owned by address 0x492Af743654549b12b1B807a9E0e8F397E44236E are being redeemed by 0x2d03B6C79B75eE7aB35298878D05fe36DC1fE8Ef.
 
 /** 
 * Burn.
 * @dev `Burn` MUST trigger when the bonds are being redeemed via staking (or being invalidated) by the bank contract.
 * @dev `Burn` MUST trigger when Bonds are burned. This SHOULD not include zero value burning.
```

Comment:
Could you also explain what "zero value burning" means in this contex?

Reply:
it means when the amount corresponding to the given class and nonce of the bonds is Zero. meaning we should not call functions with null value.